### PR TITLE
Removed Firefox Preview

### LINKED ISSUE #162: Remove Firefox Preview
Now that Firefox Preview has been released, that topic is no longer needed. It needs to be removed from the UI and from the user storage.

### PR DESCRIPTION
**Pull request by WesleyBranton**
This pull request removes Firefox Preview from the list of products to monitor, since Firefox Preview was released into the regular Firefox Android forum. Closes #162.

**WAIT TO PUSH THIS INTO RELEASE. Currently, the Firefox Android Beta questions are sent to the Firefox Preview forum category. Until that topic is merged into the regular Firefox Android section by the SUMO developers, this pull request should be excluded from the release.**

### CODE REVIEW

**Comment by dannycolin** [comment]
I think we could revive this PR and merge it when ready :)

**Comment by jhonatasrm** [comment]
I think we can create a new version this weekend and contemplate that removal as well or wait for more improvements.

**Comment by WesleyBranton** [comment]
I've resolved the conflicts in this commit. It can be merged now.

File: src/html/preferences.html [hunk 1]
```diff
@@ -78,7 +78,11 @@ <h3 data-i18n="products"></h3>
                 <input id="mobile" name="chooseProduct" type="checkbox" value="mobile">
                 <span data-i18n="product_mobile"></span>
             </label>
-            <label for="ios">
+            <label for="firefox-preview" class="checkbox">
+                <input id="firefox-preview" name="chooseProduct" type="checkbox" value="firefox-preview">
+                <span data-i18n="product_firefox-preview"></span>
+            </label>
+            <label for="ios" class="checkbox">
```
Comment (dannycolin) [inline src/html/preferences.html:85]: I think you accidentally readded the checkbok when rebasing your branch. Screenshot of what it looks like when I run it locally

image

**Review by dannycolin — CHANGES_REQUESTED** [review]

File: src/html/preferences.html (hunk 1 above)
Comment (WesleyBranton) [reply src/html/preferences.html]: That's what I get for using the GitHub website instead of the desktop application 😆

**Review by dannycolin — APPROVED** [review]
Looks good now!

**Pull request merged**